pkg/types/source-config.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+// SourceConfig is a 1:1 mapping for flagd SourceConfig type. JSON encoded SourceConfig becomes a startup argument for
+// flagd
+// NOTE - once we have API stability, make this a dependency at flagd to avoid duplication
+type SourceConfig struct {
```

Comment:
Minor - needs to bump this to match the latest changes [1] :) 

[1] - https://github.com/open-feature/flagd/blob/main/core/pkg/runtime/from_config.go#L51